base_input_topic: "/front_camera/image_raw"
face_detected_image_topic: "/face_detector/raw_image"
display_original_image: false
display_tracking_image: false
